src/components/DataSamplesTable.tsx

```diff
@@ -93,6 +100,30 @@ const DataSamplesTable = ({
     };
   }, [connection, recordingDialogDisclosure]);
 
+  const [recordingOptions, setRecordingOptions] = useState<RecordingOptions>({
+    continuousRecording: false,
+    recordingsToCapture: 1,
+  });
+  const handleRecord = useCallback(
+    (recordingOptions: RecordingOptions) => {
+      setRecordingOptions(recordingOptions);
+      isConnected
+        ? recordingDialogDisclosure.onOpen()
+        : connectToRecordDialogDisclosure.onOpen();
+    },
+    [connectToRecordDialogDisclosure, isConnected, recordingDialogDisclosure]
+  );
+
+  const tourStart = useStore((s) => s.tourStart);
+  useEffect(() => {
+    if (
+      !recordingDialogDisclosure.isOpen &&
+      gestures.length === 1 &&
+      gestures[0].recordings.length === 1
+    ) {
+      tourStart(TourId.CollectDataToTrainModel);
+    }
+  }, [gestures, recordingDialogDisclosure.isOpen, tourStart]);
```

Comment:
Would have prefered to do this on recording dialog close, but the latest version of the gesture recordings aren't available at that point.